sdk/py/apepay/manager.py

```diff
@@ -89,21 +91,32 @@ def _parse_validator(self) -> Callable[[_ValidatorItem], Validator]:
     def set_validators(self) -> ContractTransactionHandler:
 
         @wraps(self.contract.set_validators)
-        def order_validators(*validators: _ValidatorItem, **txn_kwargs) -> ReceiptAPI:
+        def set_validators(*validators: _ValidatorItem, **txn_kwargs) -> ReceiptAPI:
             # NOTE: Always keep sets sorted, ensure no duplicates
-            return self.contract.set_validators(
-                sorted(v.address for v in set(map(self._parse_validator, validators))),
-                **txn_kwargs,
-            )
+            new_validators = sorted(v.address for v in set(map(self._parse_validator, validators)))
+            # NOTE: Okay to use Click here, intended primarily to support interactive features
+            click.echo("Validators update:")
+            for line in Differ().compare(tuple(v.address for v in self.validators), new_validators):
+                click.echo(line)
+            return self.contract.set_validators(new_validators, **txn_kwargs)
 
-        return cast(ContractTransactionHandler, order_validators)
+        return cast(ContractTransactionHandler, set_validators)
 
     def add_validators(self, *new_validators: _ValidatorItem, **txn_kwargs) -> ReceiptAPI:
         return self.set_validators(
             *(set(self.validators) | set(map(self._parse_validator, new_validators))),
             **txn_kwargs,
         )
 
+    def replace_validator(self, old_validator, new_validator, **txn_kwargs) -> ReceiptAPI:
+        return self.set_validators(
```

Comment:
How come this method doesn't take already parsed `_ValidatorItem`s like other alike methods, such as `set_validator()`

Reply:
9bbd0e5